src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Extensions/SyntaxTreeExtensions.vb

```diff
@@ -325,8 +326,17 @@ recurse:
 
         <Extension()>
         Public Function IsInPreprocessorDirectiveContext(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As Boolean
+            Dim directive As DirectiveTriviaSyntax = Nothing
+            Return IsInPreprocessorDirectiveContext(syntaxTree, position, cancellationToken, directive)
+        End Function
+
+        Friend Function IsInPreprocessorDirectiveContext(
```

Comment:
Did anything use this?

Reply:
yes.  used above, and used in teh new helper i added :)